### ISSUE TITLE
GLS program activity report

### ISSUE THREAD

**Issue by andkov** [dss-hmi/suicide-prevention-2019#1]
We receiving a program activity summary from Kim as a spreadsheet. Create a report (series of reports) that describe where and how much the program was administered in selected counties.

**Comment by Jodh** [comment]
Do you want the sheets to be csv and individual files? Instead of one whole worsksheet.

**Comment by andkov** [comment]
@Jodh , no. Kim is working with that excel file.  She adds rows as new program activities are measured (training and community awareness). We want to keep this excel file as immutable as possible. This is out line of communication to Kim's program performance data. 

The script `./manipulation/0-greeter-gls.R`  establishes the data input of this excel sheet into out environment, conducts basic grooming, and stores the results in `./data-unshared/derived/` as an `.rds` and `.csv` form.  This object becomes a source of data for subsequent analytic efforts involving GLS activity. 

Will create a few issues in a minute with suggestions what you can work on as you study this script.

**Comment by andkov** [comment]
@Jodh , study the ggplot examples in this script and the create a ggplot:

- X = bins of time (year, quarter, month?)
- Y = some measure (N of people trained) 
- audiences are likely to be treated differently
- county-by-region
- we want to understand the temporal pattern of training delivery (which counties when?)
-  how do counties differ in timing and amount  of received training?
